### ISSUE TITLE
Cannot modify rows in dashboard table editor when using composite primary key

### ISSUE THREAD

**Issue by officiallymarky** [supabase/supabase#2769]
# Bug report

Cannot modify rows in dashboard table editor when using composite primary key

## Describe the bug

When using composite primary key, I get an error "can't find primary key" when trying to edit in the table editor.  This is the same error when you have a table with no primary key.

## To Reproduce

Create a composite primary key and try to edit in dashboard table editor

## Expected behavior

The ability to edit rows that use a composite primary key

**Comment by kiwicopple** [comment]
Good catch @officiallymarky - we'll fix this in `supabase/grid`. I'll transfer the issue there now
